### ISSUE TITLE
Airpurifier 2S Model info: Unknown

### ISSUE THREAD

**Issue by schurgan** [aholstenson/miio#262]
Hi,
I do not understand why my Airpurifier 2S is not recognized. I have the latest version miio@0.15.6 on it. The following is displayed:

Device ID: 131413894
Model info: Unknown
Address: 192.168.178.64
Token: ???
Support: Unknown

Please help

**Comment by schurgan** [comment]
Nobody knows what you can do?  Is there anyone here?

**Comment by dottoreFell** [comment]
What is firmware version of Your device? Is it CN version or global, EU? Does it work normally with Mi Home app?

**Comment by schurgan** [comment]
1.4.3_21106 EU. Normally work with MiHome

**Comment by heccoder** [comment]
Same issue here, I used to be able to get a token this way for my Airpurifier 2S, not anymore.

**Comment by dottoreFell** [comment]
@sonoseco with last firmware versions You can't get token with miio. Easiest way as for now seem to be installing older MiHome version that exposes token in logs.

@schurgan I have the same firmware and it is working fine. Did You try using it with token?

**Comment by schurgan** [comment]
Yes I have.  It works only with the plugin Airpurifier-domoticz.  Without it does not work
